civis/io/_databases.py

```diff
@@ -41,7 +46,12 @@ def query_civis(sql, database, api_key=None, credential_id=None,
     >>> run = query_civis(sql="DELETE schema.table", database='database')
     >>> run.result()  # Wait for query to complete
     """
-    client = APIClient(api_key=api_key)
+    if api_key is not None:
+        warnings.warn('The "api_key" parameter is deprecated and will be '
+                      'removed in v2. Please use the `client` parameter '
+                      'instead.', FutureWarning)
+    if client is None:
+        client = APIClient(api_key=api_key, resources='all')
```

Comment:
We shouldn't use `resources='all'` in a default position.  In CivisFuture we have to for pubnub, but here we ought to use `resources='base'`.  The long term fix is to get the required endpoints for CivisFuture into the base resources.